dev_guide/templates.adoc

```diff
@@ -12,35 +12,40 @@ toc::[]
 == Overview
 A template describes a set of
 link:../architecture/core_objects/overview.html[resources] that can be
-customized and processed to produce a configuration. Each template is a
-parameterized list that is used by OpenShift to create a list of resources,
-including services, pods, routes, buildconfigs, etc. A template also defines a
-set of labels to apply to every resource created by the template.
+parametrized and processed to produce a list of resources that are ment to be
+created by OpenShift.
+The list of resource can include services, pods, routes, buildconfigs, etc.  A
+template also defines a set of labels to apply to every resource created by the
```

Comment:
s/every resource created by the template/every resource defined in the template/

created here isn't a good match, as template don't create objects.